share/functions/__fish_preview_current_file.fish

```diff
@@ -0,0 +1,16 @@
+function __fish_preview_current_file --description "Open the current file in a pager"
+    # set pager to user-preferred pager or less
+    set -l pager "less --"
+    set -q PAGER && echo $PAGER | read -at pager
+
+    # get current or last token, use eval to resolve $HOME and ~
+    set -l file (eval echo (commandline -t))
```

Comment:
I think `eval set -l file (commandline -t)` does the same thing.
Since a token like `a{1,2}` can expand to multiple tokens, maybe it's better to explicitly pick the first one for now? That'd be `set -l file (eval echo (commandline -t))[1]`. Or open all of them in less?

Reply:
I ignored the `a{1,2}` after trying for a while. I did not get it to work for both, this kind of expansion as well as single filenames containing whitespace. If this is possible, please give me an example.